plugin/src/main/java/org/opennms/velocloud/connections/ConnectionManager.java

```diff
@@ -111,16 +112,23 @@ public Connection addConnection(final String alias, final String orchestratorUrl
         return connection;
     }
 
+    /**
+     * Attempts to connect a client with the given credentials to the given orchestrator
+     *
+     * @throws VelocloudApiException    If the credentials are invalid
+     */
+    public void validateConnection(String alias) throws VelocloudApiException {
```

Comment:
Shouldn't this be a `ConnectionValidationError`?

Reply:
Looking at the way we use this, this should just return something instead of throwing. Maybe we can just return the Exception or `null` if it is valid?